clients/cli/nmcli.c

```diff
@@ -676,6 +890,8 @@ nmc_cleanup (NmCli *nmc)
 		nmc->pager_pid = 0;
 	}
 
+	g_free (nmc->color_scheme);
```

Comment:
let's not leave dangling pointers in the global struct. E.g. use `nm_clear_g_free()`.

Reply:
Ok.